package.json

```diff
@@ -19,13 +19,11 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
+    "eject": "react-scripts eject",
+    "format": "prettier --write ./src",
```

Comment:
There are 4 scripts being added here. 2 make updates using eslint and prettier and 2 just perform checks and return either a 1 or 0 (success or failure)